pkg/systemd/hw-detect.js

```diff
@@ -61,6 +63,60 @@ function findPCI(udevdb, info) {
     }
 }
 
+function findMemoryDevices(udevdb, info) {
+    const memoryArray = [];
+    const dmipath = '/devices/virtual/dmi/id';
+    if (!(dmipath in udevdb)) {
```

Comment:
Style issue: You don't need { } wrappers for a single line. The code further below doesn't do it either.